packages/pluggableWidgets/shrinking-header-web/src/ShrinkingHeader.tsx

```diff
@@ -0,0 +1,83 @@
+import { createElement, ReactElement, useCallback, useEffect, useState } from "react";
+
+import { ShrinkingHeaderThreshold } from "./components/ShrinkingHeaderThreshold";
+import { ShrinkingHeaderLinear } from "./components/ShrinkingHeaderLinear";
+
+import { ShrinkingHeaderContainerProps } from "../typings/ShrinkingHeaderProps";
+
+export default function ShrinkingHeader(props: ShrinkingHeaderContainerProps): ReactElement | null {
+    const {
+        name,
+        tabIndex,
+        style,
+        class: className,
+        content,
+        threshold,
+        shrinkThreshold,
+        initHeight,
+        shrunkHeight
+    } = props;
+
+    const actualShrinkThreshold = shrinkThreshold.value ? Number(shrinkThreshold.value.toString()) : undefined;
+
+    const [shrinkingHeaderThresholdElement, setShrinkingHeaderThresholdElement] = useState<HTMLDivElement>();
+
+    const setShrinkingHeaderThreshold = useCallback(
+        (node: HTMLDivElement | null) => {
+            setShrinkingHeaderThresholdElement(node ?? undefined);
+        },
+        [setShrinkingHeaderThresholdElement]
+    );
+
+    /*
+        Position "sticky" behaviour breaks when navigating from another page in the Mendix application.
+        This is due to the css rule `overflow-x: hidden` which is applied to `div class="mx-placeholder" />`.
+        This useEffect iterates through all ancestor elements and removes this rule, except for the body element.
+     */
```

Comment:
An `overflow-x: hidden` rule on any parent element except the body causes sticky positioned elements to break. This code is pretty limited in that it only removes inline styles.
To improve this situation I will add a note about the overflow issue in the upcoming documentation, so that users can anticipate on that. Besides that, I will simply resolve the `mx-placeholder` issue by adding the following to the widget styles:
```
.mx-placeholder {
  overflow-x: visible !important;
}
```